sys/compat/cheriabi/cheriabi_misc.c

```diff
@@ -1897,9 +1895,11 @@ cheriabi_ptrace(struct thread *td, struct cheriabi_ptrace_args *uap)
 	case PT_VM_ENTRY:
 		error = COPYOUT(&r.pve, uap->addr, sizeof r.pve);
 		break;
+#endif
 	case PT_IO:
-		error = COPYOUT(&r.piod, uap->addr, sizeof r.piod);
+		error = copyoutcap(&r.piod, uap->addr, sizeof(r.piod));
```

Comment:
In cases like this I prefer to update the `piod_len` field directly (at least I assume that's what can change) rather than copying the whole structure out.  My goal is to minimize the number of `copyoutcap` calls since each one is a potential capability leak that requires ongoing auditing.

Reply:
Ok.